run_inrerence.py

```diff
@@ -39,24 +39,39 @@ def parse_args():
         type=float,
         default = 0.2
     )
-
+    parser.add_argument(
+        "--classfile", "-cf",
+        help="Path to json file to load classes and colors",
+        required=True,
+        type=str
+    )
     args = parser.parse_args()
 
     return args
 
 
-def writeConfidence(img, val, x, y, class_name):
+def writeConfidence(img, val, x, y, colors, class_name):
     pos = (x, y - 20)
     col = colors[class_name]
     str_val = "{:.2f}".format(val)
     cv2.putText(img, str_val, pos, cv2.FONT_HERSHEY_SIMPLEX, 1, col, 2)
 
 
+def read_colors_from_json(filename):
```

Comment:
```suggestion
  def get_config(filename=None):
```

and add a default config with a single class `{"id": 1, "name": "default", "color": (0,0,0)}`